ui/src/app/model/auto-gen.ts

```diff
@@ -0,0 +1,8 @@
+export default interface ICreateSequence {
+    Id?: string
+    Name: string
+    SequenceKind: string
+    SkipRangeMin?: string
```

Comment:
Why string for the ranges and start values and not integer?

Reply:
Because if someone does not set it, then the default value will be 0, which is incorrect. hence the value will be parsed in backend to check if it is unset or set.

Reply:
I see, thanks! Does the component not have a method indicating is the value was set/unset? Also when someone actually sets the value 0 - they will be allowed or there is validation? I saw a regex somewhere but that just ensures numeric, which means 0 would be allowed - please correct if am wrong there.

Reply:
Yes, 0 is a valid value. The method to check if it is set or unset is part of the backend code which skips this property if the string is empty